### PR TITLE
Revert similarity metric and schema to ES2 versions

### PR DESCRIPTION
**Pull request by barrucadu**
We have a problem with search results being Weird.  These changes fix it.  We'll need to revisit this before moving to ES6 though.

---

[Trello card](https://trello.com/c/C3STgB2P/86-ordering-of-search-results-is-weird)

### CODE REVIEW

File: spec/unit/elasticsearch_index_spec.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
   end
 
   it "returns nil for real_name when elasticsearch returns a 404 response" do
-    stub_request(:get, "http://example.com:9200/government_test/_alias")
```
Comment (brucebolt) [inline spec/unit/elasticsearch_index_spec.rb]: Do we need to revert the change from `_aliases` to `_alias` (which is deprecated in ES 6.x)?  I assume it doesn't affect the search results.

Comment (barrucadu) [reply spec/unit/elasticsearch_index_spec.rb]: Probably not, I just reverted the whole commit.

Comment (brucebolt) [reply spec/unit/elasticsearch_index_spec.rb]: It's probably a good idea to leave this as `_alias`, since it'll be one less thing to fix when we get round to upgrading to ES6.

**Review by brucebolt — APPROVED** [review]
